src/autoplex/auto/rss/flows.py

```diff
@@ -19,24 +21,32 @@ class RssMaker(Maker):
     ----------
     name: str
         Name of the flow.
-    path_to_default_config_parameters: Path | str | None
-        Path to the default RSS configuration file 'rss_default_configuration.yaml'.
-        If None, the default path will be used.
+    config_file: Path | str | None
+        Path to the custom configuration file that defines the setup parameters for the whole RSS workflow.
+        If not provided, the default file 'rss_default_configuration.yaml' will be used.
     """
 
     name: str = "ml-driven rss"
-    path_to_default_config_parameters: Path | str | None = None
+    config_file: Path | str | None = None
+    CONFIG = loadfn(MODULE_DIR / "rss_default_configuration.yaml")
```

Comment:
Would the approach work if the RSS Maker was initialised during another Job?

Reply:
Hi @JaGeo , do you mean something like this ?

```
@job
def rss_check(test_dir):


    job1 = RssMaker()
    job1 = job1.make()

    return job1
```

Reply:
Yes

Reply:
The current implementation will not work if the custom YAML path is provided inside the nested job and this job is executed on a different machine 😅 

But I cannot think of such a use case yet.

Reply:
#312 ?

Reply:
I meant for RssMaker use case with custom yaml. I am not sure I get your comment here

Reply:
Yes, sure. But the situation is similar. If the RSSmaker is part of a larger workflow, it will not work anymore

Reply:
Maybe you can add the warning about the config file limits (as you put it in the docs) in the docstring as well?

Reply:
Irrelevant now